tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,70 @@
+import os
+from threading import Thread
+import dotenv
+import pytest
+from selenium import webdriver
+from dotenv import load_dotenv
+import requests
+
+from todo_app.flask_config import Config
+from todo_app.app import create_app
+
+
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Chrome("./chromedriver") as driver:
+        yield driver
+
+# Id organisation board on trello - create default id
+#
+def create_trello_board():
+    config = Config()
+    response = requests.post(
+        url=f'{config.TRELLO_BASE_URL}/boards',
+        params={
+            'key': config.TRELLO_KEY,
+            'token': config.TOKEN,
+            'name': 'Selenium Test Board'
+        }
+    )
+    print (response.text)
+    return response.json()['id']
+
+
+def delete_trello_board(board_id):
+    config = Config()
+    requests.delete(
+        url=f'{config.TRELLO_BASE_URL}/boards/{board_id}',
+        params={
+            'key': config.TRELLO_KEY,
+            'token': config.TOKEN,
+        }
+    )
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Load variables?
+    load_dotenv(override=True)
+
+    # Create the new board & update the board id environment variable
+    board_id = create_trello_board()
+    os.environ['BOARD'] = board_id
```

Comment:
Here you set the BOARD environment variable, but you don't override the LIST or COMPLETELIST environment variables, so the test environment still loads your real (or at least, development) cards.

In order to prevent that, we want to also override the list environment variables, e.g. we could add a function something like:
```python
def override_list_ids(board_id):
    # Get the lists our new board has created:
    config = Config()
    lists = requests.get(
        url=f'{config.TRELLO_BASE_URL}/boards/{board_id}/lists',
        params={
            'key': config.TRELLO_KEY,
            'token': config.TOKEN,
        }).json()
    # Match them up and replace the relevant environment variables with the new list ids
    for list in lists:
        if list['name'] == "To Do":
            os.environ['LIST'] = list['id']
        if list['name'] == "Done":
            os.environ['COMPLETELIST'] = list['id']
```
We could then call that function after setting the board id, and it should resolve that for us